### DISCUSSION TITLE
Basic auth with redirect

### DISCUSSION THREAD

**Discussion by Jak-Tan** [cypress-io/cypress#14395] [Questions and Help]
I'm testing an application in which I need to provide basic auth headers after a redirect. I'm making a request to url1 which eventually redirects to url2 however the auth headers are not being passed and I'm getting a 401. Is there a solution for this? 

```
cy.request({
      url: `/url1.com`,
      headers: {
        auth: {
          username: 'blah',
          password: 'blah',
        },
      },
    });
```

**Comment by anand-naik** [comment]
Similar issue is observed in our case as well. 
For our application, during login process using WNA (Windows Native Authentication), there are multiple redirections (302). Cypress is not able to enter credentials using Basic Auth.
